app/assets/javascripts/references.js

```diff
@@ -1,4 +1,61 @@
 $(document).ready(function() {
+  var resultsElement = $('#crossref-results');
+
+  var hideResults = function() {
+    resultsElement.toggleClass('hidden', true);
+    resultsElement.children().remove();
+  }
+
+  var hideResultsAndClearSubmittable = function() {
+    hideResults();
+    $('.crossref').toggleClass('submittable', false);
+    $('#crossref-locator-id').val('');
+  }
+
+  var showCrossrefResults = debounce(function() {
+    if (this.value === '') {
+      hideResultsAndClearSubmittable();
+    } else {
+      $.get('https://search.crossref.org/dois?sort=score&q=' + this.value).done(function(data) {
```

Comment:
you can also search using the API
http://api.crossref.org/funders/100000015/works?query=global+state&filter=has-orcid:true

was it easier to use search.crossref?